doc/man1/s_client.pod

```diff
@@ -209,6 +220,34 @@ be used.
 
 The private format to use: DER or PEM. PEM is the default.
 
+=item B<-cert_chain>
+
+A file containing trusted certificates to use when attempting to build the
+client certificate chain when certificate specified via B<-cert> option is in
+use.
+
+=item B<-build_chain>
+
+Specify whether the application should build the certificate chain to be
+provided to server.
+
+=item B<-xkey infile>, B<-xcert infile>, B<-xchain>
+
+Specify an extra certificate, private key and certificate chain, these behave
```

Comment:
I'd end the sentence after "certificate chain" and start a new one at "These behave".